mount/mount.go

```diff
@@ -22,3 +28,48 @@ func All(mounts []Mount, target string) error {
 	}
 	return nil
 }
+
+// MountsOption are option for mount. It indicate that we will mount Mounts to TmpParentDir/TmpDirPrefix
+// for TmpParentDir and TmpDirPrefix, please see ioutil.TempDir.
+type MountsOption struct {
+	Mounts       []Mount
+	TmpParentDir string
+	TmpDirPrefix string
+}
+
+// VisitMountedDir mounts all the provided mounts to the tmp dir, and pass the tmp dir to f.
```

Comment:
Clarify that the mount is undone at the end too?

Reply:
done